python/friesian/example/recall/search.py

```diff
@@ -85,14 +85,9 @@ def do_search(partition):
 def search(args):
     set_env(args.num_threads)
     if args.cluster_mode == "yarn":
-        num_executors = 12
-        executor_cores = args.num_threads
-        executor_memory = "12g"
-        driver_cores = 4
-        driver_memory = "4g"
-        sc = init_orca_context("yarn", cores=executor_cores,
-                               num_nodes=num_executors, memory=executor_memory,
-                               driver_cores=driver_cores, driver_memory=driver_memory,
+        sc = init_orca_context("yarn", cores=args.num_threads,
+                               num_nodes=args.num_tasks, memory=args.memory,
+                               driver_cores=args.num_threads, driver_memory='16g',
```

Comment:
can we just use the default value of driver cores and driver memory?

Reply:
ok